src/lib/three.js

```diff
@@ -23,7 +23,7 @@ require('../../node_modules/three-dev/examples/js/loaders/OBJLoader');  // THREE
 require('../../node_modules/three-dev/examples/js/loaders/MTLLoader');  // THREE.MTLLoader
 require('../../node_modules/three-dev/examples/js/loaders/ColladaLoader');  // THREE.ColladaLoader
 require('../../vendor/Raycaster');  // THREE.Raycaster
-require('../../node_modules/three-dev/examples/js/controls/VRControls');  // THREE.VRControls
-require('../../node_modules/three-dev/examples/js/effects/VREffect');  // THREE.VREffect
+require('../../node_modules/webvr-libs/VRControls');  // THREE.VRControls
```

Comment:
when this is merged upstream and three-dev is bumped, we'll point back to @ngokevin's